src/main/resources/static/css/style.css

```diff
@@ -394,7 +395,7 @@ a {
 	display: inline-block;
 	text-transform: uppercase;
 	bottom: 40px;
-	margin: calc(100vh - 220px) 0 0 0;
+	margin: calc(80vh - 220px) 0 0 0;
```

Comment:
This was only a miscalculation, see previous class definition: `max-height: 80vh;`
![Screenshot 2020-07-01 01 12 34](https://user-images.githubusercontent.com/60111772/86296100-4d37af80-bbf8-11ea-8b3a-f87c9ff9a9cc.png)

Reply:
Misunderstood context of this `margin`. Newest commit changes it back to original.